### ISSUE TITLE
brew cask install Udeler

### ISSUE THREAD

**Issue by muescha** [FaisalUmair/udemy-downloader-gui#14]
you should add udeler as a brew cask formula for easy install

**Comment by FaisalUmair** [comment]
By easy install do you mean building from the source on github for any unreleased version because all the released packages are already in the easiest form, you simply download and install.

**Comment by muescha** [comment]
when there is a formula by `brew cask`
then all is done by brew with one line `brew cask install udeler` 
see https://caskroom.github.io/

**Comment by FaisalUmair** [comment]
Ok, I will check it.

Thanks

**Comment by FaisalUmair** [comment]
A cask for Udeler is now available

`brew cask install udeler`

**Comment by FaisalUmair** [comment]
I was thinking about making a gui for Homebrew Cask. I have used homebrew before, but didn't knew much about cask. 

While I was searching, I found your comments on few suggestions that were already raised about this. I think that was in 2013, now Homebrew Cask has grown a lot over time. All the packages available on Homebrew Cask could help in making an app that helps you discover other apps and hence resulting in a unofficial mac app store.
